src/tokens/split_spaces.c

```diff
@@ -0,0 +1,78 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   split_spaces.c                                     :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: mcourtoi <mcourtoi@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/03/27 20:50:26 by mcourtoi          #+#    #+#             */
+/*   Updated: 2023/03/27 21:06:38 by mcourtoi         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "minishell.h"
+
+int	split_is_null(t_token_lst *const token_lst,
+		t_token **const token, t_str const *const tmp)
+{
+	if (ft_strlen((*token)->str))
```

Comment:
I assume that you are checking whether `(*token)->str` is an empty string. If so, just checking `*(*token)->str` instead of calling `ft_strlen()` would suffice, and it always takes  1 instruction instead of N.

Reply:
No. (*token)->str is strdup(""); so it won't works to just check whether it exists or not.